WTWSMS/common/scripted_triggers/ACR_scripted_triggers.txt

```diff
@@ -83,6 +88,7 @@ is_ancient_religion_trigger = {
 		religion = hellenic_pagan
 		religion = egyptian_pagan
 		religion = aten_pagan
+		religion = assyrian_pagan
```

Comment:
Currently it is though, unless you intend to reuse it, in which case I suggest to create a new one.